contrib/gitian-descriptors/gitian-osx-signer.yml

```diff
@@ -34,5 +34,5 @@ script: |
 
   tar -xf ${UNSIGNED}
```

Comment:
Here is a changeset from bitcoinxt, further up in this file:

-- "url": "https://github.com/bitcoin/bitcoin-detached-sigs.git"
+- "url": "https://github.com/bitcoinxt/bitcoin-detached-sigs.git"